docs/topics/proc_running-integration-tests.adoc

```diff
@@ -7,8 +7,8 @@
 //
 // Rationale: This procedure is highly similar for all boosters and mission except the Secured Booster.
 
-[#running-the-booster-integration-tests_{context}]
-= Running the {parameter-mission-name} Booster Integration Tests on {parameter-runtime-name}
+[#running-the-parameter-mission-booster-integration-tests_{context}]
```

Comment:
I believe _parameter_ should not be here because the `parameter` part in `{parameter-mission-name}` only denotes it is a local parameter inside the module. The key concept here is `mission`, so I think the anchor should be `running-the-mission-booster-integration-tests_{context}`, or even `running-the-booster-integration-tests_{context}`, as it was.